cinema/views.py

```diff
@@ -120,7 +120,7 @@ def upload_image(self, request, pk=None):
         item = self.get_object()
         serializer = self.get_serializer(item, data=request.data)
 
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
```

Comment:
you don't need if and response with 400 status, serializer does it automatically